src/main/java/zd/zero/waifu/motivator/plugin/alert/AlertAssetProvider.java

```diff
@@ -1,14 +1,17 @@
 package zd.zero.waifu.motivator.plugin.alert;
 
-import com.fasterxml.jackson.databind.MapperFeature;
-import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.gson.Gson;
```

Comment:
Gson is bundled with all the platforms, a least all the platforms I know about (have not had issues with the Doki-Theme)

Reply:
That's nice, with this it makes more sense to drop the Jackson if platforms already use GSON. I once had a problem with another library which is included in the classpath like `FilenameUtils`, it works in IntelliJ but doesn't work in Webstorm see #84. So I'll just build the plugin locally and test it to different platforms.